starknet_devnet/starknet_wrapper.py

```diff
@@ -135,13 +143,62 @@ async def initialize(self):
             await self.fee_token.deploy()
             await self.accounts.deploy()
             await self.__deploy_chargeable_account()
-            await self.__predeclare_oz_account()
+            await self.__predeclare_starknet_cli_account()
             await self.__udc.deploy()
 
             await self.__preserve_current_state(starknet.state.state)
-            await self.create_empty_block()
+            await self.__create_genesis_block()
             self.__initialized = True
 
+    async def __create_genesis_block(self):
+        """Create genesis block"""
+        transactions: List[DevnetTransaction] = []
+        transaction_hash = 1
+
+        # Declare transactions
+        declare_hashes = [
+            to_bytes(FeeToken.HASH),
+            to_bytes(UDC.HASH),
+            ChargeableAccount(self).class_hash_bytes,
```

Comment:
Can't this be accessed with `self.config.account_class.hash_bytes`?

Reply:
fixed in 69d87a2326caf32e7640e0df842f6991a81137fe